docs/sources/setup/migrate/ssd-to-distributed/_index.md

```diff
@@ -0,0 +1,372 @@
+   * Gradually shifted to distributor component
+   * Both write paths share the same object storage, ensuring data consistency
+2. Read Path (`/loki/api/v1/query`):
+   * Routes to either SimpleScalable read or Distributed query-frontend
+   * Query results are consistent since both architectures read from same storage
+3. Admin/Background Operations:
+   * Compaction, retention, and rule evaluation handled by either backend or respective distributed components
+   * Operations are coordinated through object storage locks
+
+To start the migration process:  
+
+1. Create a copy of your existing `values.yaml` file and name it `values-migration.yaml` (You can also make these changes within your current values file).
+
+    ```bash
+    cp values.yaml values-migration.yaml
+    ```
+
+1. Next modify the following parameters; `deploymentMode`, `ingester` and components based on the annotations below.
+
+   ```yaml
+   ---
+   loki:
+     schemaConfig:
+       configs:
+         - from: "2024-04-01"
+           store: tsdb
+           object_store: s3
+           schema: v13
+           index:
+             prefix: loki_index_
+             period: 24h
+     storage_config:
+       aws:
+         region: eu-central-1
+         bucketnames: aws-chunks-bucket
+         s3forcepathstyle: false
+     ingester:
+       chunk_encoding: snappy
+       # Add this to ingester; this will force ingesters to flush before shutting down
+       wal:
+         flush_on_shutdown: true
+     ruler:
+       enable_api: true
+       storage:
+         type: s3
+         s3:
+           region: eu-central-1
+           bucketnames: aws-ruler-bucket
+           s3forcepathstyle: false
+         alertmanager_url: http://prom:9093
+     querier:
+       max_concurrent: 4
+
+     storage:
+       type: s3
+       bucketNames:
+         chunks: "aws-chunks-bucket"
+         ruler: "aws-ruler-bucket"
+       s3:
+         region: eu-central-1
+
+   # Important: Make sure to change this to SimpleScalable<->Distributed
+   deploymentMode: SimpleScalable<->Distributed
+
+   # SSD
+   backend:
+     replicas: 2
+   read:
+     replicas: 3
+   write:
+     replicas: 3
+
+   # Distributed Loki
+   # Spin up the distributed components
+   ingester:
+     replicas: 3
+     zoneAwareReplication:
+       enabled: false
+   querier:
+     replicas: 3
+     maxUnavailable: 0
+   queryFrontend:
+     replicas: 2
+     maxUnavailable: 0
+   queryScheduler:
+     replicas: 2
+   distributor:
+     replicas: 2
+     maxUnavailable: 0
+   compactor:
+     replicas: 1
+   indexGateway:
+     replicas: 2
+     maxUnavailable: 0
+   ruler:
+     replicas: 1
+     maxUnavailable: 0
+
+   # Single binary Loki
+   singleBinary:
+     replicas: 0
+
+   minio:
+     enabled: false
+   ```
+
+   Here is a breakdown of the changes:
+   * `ingester.wal.flush_on_shutdown: true`: This will force the ingesters to flush before shutting down. This is important to ommitt data loss.
```

Comment:
```suggestion
   * `ingester.wal.flush_on_shutdown: true`: This will force the ingesters to flush before shutting down. This is important to prevent data loss.
```